docs/references.bib

```diff
@@ -2152,3 +2152,35 @@ @article{droogers2002
 	url = {https://www.scopus.com/inward/record.uri?eid=2-s2.0-0036464359&doi=10.1023%2fA%3a1015508322413&partnerID=40&md5=7322aaa4c6874878f5b1dab3c73c1718},
 	type = {Article}
 }
+
+@article{article,
+    author = {Addor, Nans and Nearing, Grey and Prieto, Cristina and Newman, A. and Le Vine, Nataliya and Clark, Martyn},
+    year = {2018},
+    month = {11},
+    pages = {},
+    title = {A Ranking of Hydrological Signatures Based on Their Predictability in Space},
+    journal = {Water Resources Research},
+    doi = {10.1029/2018WR022606}
+}
+
+@article{article,
+    author = {Clausen, B and Biggs, Barry},
+    year = {2000},
+    month = {11},
+    pages = {184-197},
+    title = {Flow variables for ecological studies in temperate streams: Groupings based on covariance},
+    volume = {237},
+    journal = {Journal of Hydrology},
+    doi = {10.1016/S0022-1694(00)00306-1}
+}
+
+@article{article,
+    author = {Olden, Julian and Poff, N.},
+    year = {2003},
+    month = {03},
+    pages = {101 - 121},
+    title = {Redundancy and the Choice of Hydrologic Indices for Characterizing Stream Flow Regimes},
+    volume = {19},
+    journal = {River Research and Applications},
+    doi = {10.1002/rra.700}
+}
```

Comment:
The article codes (`article`) need to be unique entries here, e.g. `addor2018`, `clausen2000`, `olden2003`.